src/components/combobox/combobox.tsx

```diff
@@ -867,11 +874,7 @@ export class Combobox implements LabelableComponent, FormComponent, InteractiveC
     chip?.setFocus();
   }
 
-  shiftActiveItemIndex(delta: number): void {
-    const length = this.visibleItems.length;
-    const newIndex = (this.activeItemIndex + length + delta) % length;
-    this.updateActiveItemIndex(newIndex);
-    // ensure active item is in view if we have scrolling
+  private activeInView = (): void => {
```

Comment:
Could you rename this to better express what it does? Currently, it reads to me like it will return a boolean if the active (item?) is in view.